src/api/procedures/modifyToken.ts

```diff
@@ -74,11 +74,6 @@ export async function prepareModifyToken(
     }
 
     this.addTransaction(tx.asset.makeDivisible, {}, rawTicker);
-  } else if (makeDivisible === false) {
-    throw new PolymeshError({
-      code: ErrorCode.ValidationError,
-      message: 'You cannot make the token indivisible',
-    });
```

Comment:
Why is this not required any more?

Reply:
Because it was never possible. `makeDivisible` is typed as `true | undefined`